src/renderer/src/components/common/Sidebar/Sidebar.tsx

```
@@ -0,0 +1,78 @@
+import { flex } from '@renderer/utils'
+import {
+  IconHome,
+  IconEditor,
+  IconPlot,
+  IconLab,
+  IconMemo,
+  IconCharacter
+} from '@renderer/design/icons'
+import { useLocation, useNavigate } from 'react-router-dom'
+
+export const NAVIGATION_DATA = [
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)